fast64_internal/f3d/f3d_material.py

```diff
@@ -2562,31 +2578,36 @@ class RDPSettings(bpy.types.PropertyGroup):
         items=enumTextDetail,
         default="G_TD_CLAMP",
         update=update_node_values_with_preset,
+        description="Changes type of LoD usage. Affects how tiles are selected based on texel magnification. Only works when G_TL_LOD is selected"
     )
     g_mdsft_textpersp: bpy.props.EnumProperty(
         name="Texture Perspective Correction",
         items=enumTextPersp,
         default="G_TP_PERSP",
         update=update_node_values_with_preset,
+        description="Turns on/off texture perspective correction"
     )
     g_mdsft_cycletype: bpy.props.EnumProperty(
         name="Cycle Type",
         items=enumCycleType,
         default="G_CYC_1CYCLE",
         update=update_node_values_with_preset,
+        description="Changes RDP pipeline configuration. For normal textured triangles use one or two cycle mode"
     )
     # v1 only
     g_mdsft_color_dither: bpy.props.EnumProperty(
         name="Color Dither",
         items=enumColorDither,
         default="G_CD_ENABLE",
         update=update_node_values_with_preset,
+        description="Applies your choice dithering type to output frambuffer"
     )
     g_mdsft_pipeline: bpy.props.EnumProperty(
         name="Pipeline Span Buffer Coherency",
         items=enumPipelineMode,
         default="G_PM_1PRIMITIVE",
         update=update_node_values_with_preset,
+        description="Changes primitive rasterization timing. For games besides SM64, N-prim is optimal"
```

Comment:
"If set to 1 primitive, effectively inserts a pipe sync after every triangle, substantially worsening the performance. Vanilla SM64 has synchronization bugs which may crash if this is not used. In a modern SM64 project with these bugs fixed, or in any other game like OoT, always use N primitive."